### ISSUE TITLE
Userscript Idea: Blame

### ISSUE THREAD

**Issue by ROpdebee** [ROpdebee/mb-userscripts#675]
Some way of facilitating finding the edits which changed something would be useful. I was imagining some new link next to each AR which opens an edit search that looks for any edit that added/changed/deleted that AR, as a way to quickly find the relevant edits without having to spend time either 1) sifting through pages upon pages of irrelevant edits, or 2) filling in the search form manually. It could also be extended to things other than ARs, like release information, tracklist, cover art, etc.

In short: git blame for MusicBrainz
